core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/backoff/SdkDefaultBackoffStrategy.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core.retry.backoff;
+
+import java.time.Duration;
+import software.amazon.awssdk.core.internal.retry.SdkDefaultRetrySetting;
+import software.amazon.awssdk.core.retry.RetryPolicyContext;
+import software.amazon.awssdk.core.retry.RetryUtils;
+
+public class SdkDefaultBackoffStrategy implements BackoffStrategy {
+
+    private final FullJitterBackoffStrategy defaultBackoffStrategy = FullJitterBackoffStrategy.builder()
+                                        .baseDelay(SdkDefaultRetrySetting.BASE_DELAY)
```

Comment:
formatting is a bit off here.  should this be `.baseDelay(SdkDefaultRetrySetting.THROTTLED_BASE_DELAY)`?

Reply:
I get the formatting question. Not sure about the second question...

Reply:
yeah, I don't know what I was thinking back then either ╮(╯_╰)╭